app/core/log.py

```diff
@@ -1,13 +1,45 @@
 import logging
 import logging.config
 import queue
+from enum import Enum
 from logging.handlers import QueueHandler, QueueListener
 from pathlib import Path
 from typing import Any
 
+import uvicorn
+
 from app.core.config import Settings
 
 
+class ModifiedFormatter(uvicorn.logging.DefaultFormatter):
+    class console_color(Enum):
```

Comment:
I *think* if `console_color` inherit from str, you won't need to call `.value` in strings.